CONTRIBUTING.md

```diff
@@ -40,7 +40,7 @@ pip install -r requirements.txt
 
 To ensure that [VSCode](https://code.visualstudio.com/) and other code editors use the correct conda environment, run the following command to add the path of the `jovian-py-dev` env to your `.bashrc` file:
 
-*Ensure that you have activated `jovian-py-dev` the environment before running the following command* 
+_Ensure that you have activated `jovian-py-dev` the environment before running the following command_
```

Comment:
activated `jovian-py-dev` the environment --> 
activated `jovian-py-dev` environment 

I think the *the* can be dropped..

Reply:
Thanks will do.